nss-day-cohort-51/nutshell-vampire-brood
# Update README file with Team Details, planning, and Application operation details

# Description: Update README file with Team Details, planning, and Application operation details

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions
On Github .com, check that the README.md has new subsections added as follows:
- The Vampire Brood
- How Nutshell Works in a Nutshell
- Entity Relationship Diagram (with image file)
- Wireframe, Component View (with image file)

# Checklist:
